src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/IllegalTokenCheck.java

```diff
@@ -51,6 +51,17 @@
  * <pre>
  * &lt;module name=&quot;IllegalToken&quot;/&gt;
  * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * public void myTest() {
+ *     outer: // violation
+ *     for (int i = 0; i &lt; 5; i++) {
+ *         if (i == 1) {
+ *             break outer;
+ *         }
+ *     }
+ * }
+ * </pre>
  * <p>
  * To configure the check to find token LITERAL_NATIVE:
```

Comment:
`to find token LITERAL_NATIVE:`  -> to report violation on token LITERAL_NATIVE:

Reply:
ok, I will change it.